bot/src/main.rs

```diff
@@ -2662,18 +2674,28 @@ mod test {
     #[test]
     fn test_calculate_commission_at_end_of_epoch_irrelevant_history() {
         let epoch: u64 = 123;
-        let current_commission = 100.0;
+        let current_commission = 10.0;
         let expected_commission = 100.0;
 
         // Changes:
-        // 100 -> 10 10% through epoch 124
-        let history = [CommissionChangeIndexHistoryEntry {
-            commission_before: Some(0.0),
-            commission_after: expected_commission,
-            epoch,
-            epoch_completion: 50.0,
-            ..Default::default()
-        }]
+        // 100 -> 10 50% through epoch 124.
+        // 10 -> 50 60% through epoch 124. Shouldn't matter.
```

Comment:
Maybe this is already there, but it might also be good to have a test where there is a change in 123 and 124. It seems like the change in 123 should be taken based on the code.

Reply:
Yes, that case is tested in the `test_calculate_commission_at_end_of_epoch_long_history` test.